src/migration.ts

```diff
@@ -132,7 +132,7 @@ export async function compileIncludes(
   content: string,
   processedFiles: ReadonlySet<string>,
 ): Promise<string> {
-  const regex = /^--![ \t]*include[ \t]+(.*\.sql)[ \t]*$/gm;
+  const regex = /^--![ \t]*[iI]nclude[ \t]+(.*\.sql)[ \t]*$/gm;
```

Comment:
Why? Suggest we revert this:

```suggestion
  const regex = /^--![ \t]*include[ \t]+(.*\.sql)[ \t]*$/gm;
```

Reply:
Sorry I didn't mean to leave this change in. I was toying around with the idea of writing it as `--! Include ...` before I concluded that amount of flexibility isn't worth it, particularly when uncommitting will clobber the formatting anyway.